keyboards/kopibeng/mnk65_stm32/info.json

```
@@ -1,25 +1,925 @@
 {
-    "keyboard_name": "MNK65_STM32",
     "manufacturer": "kopibeng",
+    "keyboard_name": "MNK65_STM32",
     "maintainer": "kopibeng",
-    "url": "https://monokei.co/",
-    "usb": {
-        "vid": "0x4B50",
-        "pid": "0x0652",
-        "device_version": "1.0.0"
+    "bootloader": "stm32-dfu",
+    "diode_direction": "COL2ROW",
+    "features": {
+        "bootmagic": true,
+        "command": true,
+        "console": false,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true,
+        "rgblight": true
+    },
+    "indicators": {
+        "caps_lock": "A8"
     },
     "matrix_pins": {
         "cols": ["A9", "B8", "F0", "C15", "C14", "C13", "B10", "B2", "B1", "B0", "A7", "A6", "A5", "A4", "A2"],
         "rows": ["A10", "B9", "A1", "B11", "A3"]
     },
-    "diode_direction": "COL2ROW",
-    "indicators": {
-        "caps_lock": "A8"
-    },
     "processor": "STM32F072",
-    "bootloader": "stm32-dfu",
+    "rgblight": {
+        "animations": {
+            "alternating": true,
+            "breathing": true,
+            "christmas": true,
+            "knight": true,
+            "rainbow_mood": true,
+            "rainbow_swirl": true,
+            "rgb_test": true,
+            "snake": true,
+            "static_gradient": true,
+            "twinkle": true
+        },
+        "brightness_steps": 8,
+        "led_count": 14,
+        "max_brightness": 200,
+        "saturation_steps": 8,
+        "sleep": true
+    },
+    "url": "https://monokei.co/",
+    "usb": {
+        "device_version": "1.0.0",
+        "pid": "0x0652",
+        "vid": "0x4B50"
+    },
+    "ws2812": {
+        "pin": "B4"
+    },
+    "community_layouts": ["65_ansi_blocker", "65_ansi_blocker_split_bs", "65_ansi_blocker_tsangan", "65_ansi_blocker_tsangan_split_bs", "65_iso_blocker", "65_iso_blocker_split_bs", "65_iso_blocker_tsangan", "65_iso_blocker_tsangan_split_bs"],
     "layouts": {
```

Not backwards compatible without adding an alias, which would then require it to go via `develop`
```suggestion
    "layout_aliases": {"LAYOUT": "LAYOUT_all"},
    "layouts": {
```